src/app/components/headerbar/headerbar.ui

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="HeaderBarWidget" parent="AdwBin">
+    <child>
+      <object class="GtkOverlay" id="overlay">
+        <property name="hexpand">1</property>
+        <child>
+          <object class="AdwHeaderBar">
+            <property name="show-end-title-buttons">1</property>
+            <child>
+              <object class="GtkButton" id="go_back">
+                <property name="receives-default">1</property>
+                <property name="halign">start</property>
+                <property name="valign">center</property>
+                <property name="icon-name">go-previous-symbolic</property>
+                <property name="has-frame">0</property>
+              </object>
+            </child>
+            <child type="title">
+              <object class="AdwWindowTitle" id="title">
+                <property name="visible">0</property>
+                <property name="title" translatable="no">Spot</property>
+              </object>
+            </child>
+            <child type="end">
+              <object class="GtkButton" id="start_selection">
+                <property name="receives-default">1</property>
+                <property name="label">Select...</property>
```

Comment:
Any reason this is a text button instead of an icon with `selection-mode-symbolic`?

Reply:
Thought the intent was clearer with a label, but a full button on every page kinda feels overwhelming so you might right about going with the icon only. what do you think, is the intent of the icon clear enough?

Reply:
I think the checkmark makes sense.

Reply:
Alright! I'll just keep the one that's on stable gtk for now, seeing the unavailable icon while working is itching me :p